css/src/lib.rs

```diff
@@ -398,10 +325,38 @@ impl TryFrom<&[u8]> for Signature {
     }
 }
 
-fn take(count: usize) -> impl Fn(&[u8]) -> (&[u8], &[u8]) {
+fn take<const SIZE: usize>() -> impl Fn(&[u8]) -> (&[u8], [u8; SIZE]) {
     move |input| {
-        nom::bytes::complete::take::<usize, &[u8], nom::error::Error<&[u8]>>(count)(input)
-            .expect("Size of stream should have been guaranteed to hold the bytes")
+        let parsing =
+            nom::bytes::complete::take::<usize, &[u8], nom::error::Error<&[u8]>>(SIZE)(input)
+                .expect("Size of stream should have been guaranteed to hold the bytes");
+        let array = <[u8; SIZE]>::try_from(parsing.1)
```

Comment:
had to return a concrete instead of a reference since the array only exists here, as mentioned above the compiler will most likely elide this and not do any more copies than existed previously